backend/src/data/entities/carereeJourney.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, Column, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { User } from './user';
+
+@Entity()
+export class CarereeJournay {
+  @PrimaryGeneratedColumn('uuid')
+  id: number;
+
+  @Column({ type: 'date' })
+  startDate: string;
```

Comment:
TS is used to set type 
```suggestion
  @Column()
  startDate: Date;
```